CapabilityAgent.js

```diff
@@ -199,7 +199,7 @@ async function _keyFromSeedAndName({seed, keyName}) {
     await crypto.subtle.sign(hmacKey.algorithm, hmacKey, nameBuffer));
 
   // generate Ed25519 key from HMAC signature
-  const keyPair = await Ed25519KeyPair.generate({seed: signature});
+  const keyPair = await Ed25519VerificationKey2020.generate({seed: signature});
```

Comment:
With this change, it will now no longer work for key type 2018, is that correct? Is that something we intend to do here?

Reply:
That's correct. That's consistent with our overall rollout / switch from 2018 to 2020 in our various libraries.

Reply:
@JSAssassin I've now added backwards-compatibility convenience methods and upgrade instructions over at https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/8.